_posts/2013-08-10-companies.markdown

```diff
@@ -95,35 +96,38 @@ layout: sidebar
-| `city`                    | name of address' city.
-| `state`                   | two letter state abbreviation.
-| `zip`                     | five digit zipcode. Returned as a string to maintain leading zeros.
-| `compensations`           | all available company-wide compensations. The above compensations are the default, built-in compensations.
-| `hourly`                  | available hourly compensations
-| `name`                    | name of the compensation
-| `multiple`                | amount multiplied by the base rate of a given job to calculate total compensation per hour worked. For example, if an employee at this company makes $17.00/hour as Kindergarten Cop and works 10 hours of Overtime (which has a multiple of 1.5), he would expect receive $150.
-| `fixed`                   | available fixed compensations
-| `name`                    | name of the compensation
-| `paid_time_off`           | available types of paid time off
-| `name`                    | name of the time off
-| `first_name`              | primary signatory's first name
-| `middle_initial`          | primary signatory's middle initial
-| `last_name`               | primary signatory's last name
-| `phone`                   | primary signatory's phone number
-| `email`                   | primary signatory's email
-| `home_address`            | primary signatory's home address
-| `first_name`              | primary payroll admin's first name
-| `last_name`               | primary payroll admin's last name
-| `phone`                   | primary payroll admin's phone number
-| `email`                   | primary payroll admin's email
+| Attribute                          | Type                           | Description
+| :----------                        |:-------------                  |:-------------
+| `id`                               | Integer                        | The unique identifier of this company in the Gusto system.
+| `name`                             | String                         | The name of this company.
+| `trade_name`                       | String                         | The trade name of this company. This is an optional field.
+| `ein`                              | String                         | The employer identification number for this company.
+| `entity_type`                      | String                         | The type of the company (e.g. "LLC").
+| `company_status`                   | String                         | The status of the company, one of: "Approved", "Not Approved", "Suspended".<br/><br/>"Approved": The company is approved to run payroll with Gusto.<br/><br/>"Not Approved": The company is not approved to run payroll with Gusto. In order to run payroll, the company must contact Gusto support.<br/><br/>"Suspended": The company's Gusto account is suspended and they cannot currently run payroll. In order to unsuspend their account, the company must contact Gusto support.
```

Comment:
I hate the `<br/>`s in here as much as the next person, but it was the only thing I could find that results in the markdown table looking like this: 
<img width="871" alt="Screen Shot 2020-03-09 at 2 00 25 PM" src="https://user-images.githubusercontent.com/7913604/76252782-079d4400-620f-11ea-8346-7dac2ded9c1b.png">